### PR TITLE
Add dags for loading the chaos test clusters for HyperShift

### PR DESCRIPTION
**Pull request by chaitanyaenr**
### Description
This commit:
- Adds install and workload configs with cleanup set to
   false to be able to run the chaos tests post install and load.
- Disables 4.10 dags for HyperShift.

### CODE REVIEW

**Review by mukrishn — APPROVED** [review]
LGTM

File: dags/openshift_nightlies/scripts/install/hypershift.sh [hunk 1]
```diff
@@ -112,7 +112,7 @@ install(){
     aws s3api create-bucket --acl public-read --bucket $MGMT_CLUSTER_NAME-aws-rhperfscale-org --create-bucket-configuration LocationConstraint=$AWS_REGION --region $AWS_REGION || true
     echo "Wait till S3 bucket is ready.."
     aws s3api wait bucket-exists --bucket $MGMT_CLUSTER_NAME-aws-rhperfscale-org 
-    hypershift install $HO_IMAGE_ARG  --oidc-storage-provider-s3-bucket-name $MGMT_CLUSTER_NAME-aws-rhperfscale-org --oidc-storage-provider-s3-credentials aws_credentials --oidc-storage-provider-s3-region $AWS_REGION  --enable-ocp-cluster-monitoring --metrics-set=All
+    hypershift install $HO_IMAGE_ARG  --oidc-storage-provider-s3-bucket-name $MGMT_CLUSTER_NAME-aws-rhperfscale-org --oidc-storage-provider-s3-credentials aws_credentials --oidc-storage-provider-s3-region $AWS_REGION  --platform-monitoring All --metrics-set All
```
Comment (chaitanyaenr) [inline dags/openshift_nightlies/scripts/install/hypershift.sh:115]: Will rebase once https://github.com/cloud-bulldozer/airflow-kubernetes/pull/247 gets merged. Thanks.

Comment (mukrishn) [reply dags/openshift_nightlies/scripts/install/hypershift.sh:115]: Lets merge this first and I will rebase the other PR as that takes longer to verify.

Comment (mukrishn) [reply dags/openshift_nightlies/scripts/install/hypershift.sh:115]: This arg change should get merged, without this hypershift deploy would fail.

**Pull request merged**